src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -12,17 +12,20 @@ public class StandardOutputVisitor : XmlTestExecutionVisitor
         readonly object consoleLock;
         readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
         readonly string defaultDirectory;
+        readonly bool showProgress;
 
         public StandardOutputVisitor(object consoleLock,
                                      string defaultDirectory,
                                      XElement assemblyElement,
                                      Func<bool> cancelThunk,
-                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null)
+                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null,
+                                     bool showProgress = false)
```

Comment:
This is part of the .exe, in our own fork.  I'm not quite understanding the concern.  Can you elaborate?  Who else are you concerned is relying on this file?